scripts/speech_recognition/convert_to_tarred_audio_dataset.py

```diff
@@ -77,6 +78,10 @@
     "--manifest_path", default=None, type=str, required=False, help="Path to the existing dataset's manifest."
 )
 
+parser.add_argument(
+    "--dataset_dir", default=None, type=str, required=True, help="Path to the dataset's directory."
```

Comment:
Don't make this required, default it to empty string to be compliant with older scripts.

Reply:
Firstly, Tarred datasets don't care about any relative or absolute path - they are all flattened out. All paths are flat relative to the loaded tarfile chunk itself so I don't see the reason for this flag to exist. It has no effect since tarfile should flatten it out.